conf/mesos-site.xml

```diff
@@ -99,12 +109,12 @@
 
   <property>
     <name>mesos.hdfs.journalnode.cpus</name>
-    <value>1</value>
+    <value>0.5</value>
```

Comment:
Yes, let's make sure this is consistent with SchedulerConf. @adam-mesos, actually I would like to leave these values here. These are values that users often change. I've had a few users with larger or smaller clusters change them. If it's a value that people are often changing it makes sense to keep them in the config. We might want to add a description.